dependency-reduced-pom.xml

```diff
@@ -84,7 +84,7 @@
     <maven.compiler.source>11</maven.compiler.source>
     <sentry.version>1.7.28</sentry.version>
     <javatuples.version>1.2</javatuples.version>
-    <magpie.api.version>0.1.2</magpie.api.version>
+    <magpie.api.version>0.2.0</magpie.api.version>

```

Comment:
Can you add this file to `.gitignore` and remove it from the repo?  These keep sneaking in, thanks!

Reply:
Done